tests/ffi/tcp_max_syn_backlog/main.fmf

```diff
@@ -0,0 +1,7 @@
+summary: Check that changes to network settings inside QM doesn't change network settings
+    on Host.
+test: /bin/bash ./test.sh
+duration: 20m
```

Comment:
I still see the error, I am not sure why I still see the error in the github and not respecting this 20m

Reply:
And the weirdest thing is that I had it passed just fine even without increasing the time.
Its also wrote in the attributes that "/tests/ffi/memory (timeout)took 900 s"
My test has nothing related to memory, its /ffi/tcp_max_syn_backlog.
Yesterday I saw that it worked fine even with 10 min test run.
![Screenshot from 2024-08-08 20-57-55](https://github.com/user-attachments/assets/cedd472b-8b9a-4555-9f9a-fee5cba27024)
I've also tested it on TC and it works.
Lets see if it works now...

Reply:
I see that it works now.
Looks like some infra issues with unrelated to the test other test runs.